MinPlatformPkg/Test/Library/TestPointCheckLibNull/TestPointCheckLibNull.c

```diff
@@ -697,27 +659,35 @@ TestPointSmmReadyToBootSmmPageProtection (
 }
 
 /**
-  This service verifies the system state after Exit Boot Services is invoked.
+  This service verifies the validity of the StandaloneMm page table at StandaloneMm
+  Ready To Boot.
+
+  Test subject: StandaloneMm page table.
+  Test overview: Verify the StandaloneMm page table matches the StandaloneMm memory 
+  attribute table.
+  Reporting mechanism: Set ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT.
+                       Reports an error message upon checking.
 
   @retval EFI_SUCCESS         The test point check was performed successfully.
+  @retval EFI_UNSUPPORTED     The test point check is not supported on this platform.
 **/
 EFI_STATUS
 EFIAPI
-TestPointExitBootServices (
-  VOID
-  )
+TestPointStandaloneMmReadyToBootStandaloneMmPageProtection (
+    VOID
+)
 {
-  return EFI_SUCCESS;
+    return EFI_SUCCESS;
 }
 
```

Comment:
I've generally requested formatting not to change. This area in particular actually deviated away from the proper formatting. Can you please check it compared to the original?

Reply:
Don't know how this happened but fixed.